src/chains/filecoin/filecoin/src/ready.ts

```diff
@@ -1,9 +1,15 @@
-import type { FilecoinProvider } from "@ganache/filecoin";
```

Comment:
Why not put this whole thing into index.ts, in the same way that it is in the ethereum chain?